addons/voxel-core/engine/importers/meshes.gd

```diff
@@ -106,3 +171,14 @@ func import(source_file : String, save_path : String, options : Dictionary, r_pl
 		
 		voxel_mesh.free()
 	return error
+
+
+# gets the fraction needed to offset the origin
+# returns -1, for "no change"
+func _get_frac_for_origin(option_val: int) -> float:
```

Comment:
I see that some functions are repeated in these EditorImportPlugin files, if we do make a parent class for these EditorImportPlugins we should move common functions to there to. Or maybe move them to Voxel class if they're useful for other things?